client/app/common/less/popover.less

```diff
@@ -55,8 +55,9 @@
 
                 .back-btn {
                     position: absolute;
-                    left: 10px;
+                    left: 12px;
```

Comment:
Why don't use 13px ? :)

Reply:
Because "twelve" is the largest number with a single-syllable name in English.